span.go

```diff
@@ -64,7 +76,8 @@ func (tx *Transaction) StartSpan(name, spanType string, parent *Span) *Span {
 // span type, subtype, and action; a single dot separates span type and
 // subtype, and the action will not be set.
 func (tx *Transaction) StartSpanOptions(name, spanType string, opts SpanOptions) *Span {
-	if tx == nil {
+	if tx == nil || opts.parent.IsExitSpan() {
+		// TODO: Is this the equivalent of a noop span?
```

Comment:
Yes, "dropped spans" are no-op spans. They exist to remove the need to perform nil-pointer checks all over the place when creating spans, mostly because spans may be dropped due to the max span limit being reached